environment.yml

```diff
@@ -11,6 +11,7 @@ dependencies:
   - boost=1.76.0
   - breathe=4.32.0                # dev
   - cmake=3.21.3                  # dev
+  - cpr=1.7.2
```

Comment:
Should be a `#dev` dependency, I think, since it is header only.

Reply:
It's not header-only.